test/msw-api/handle-stream.test.ts

```diff
@@ -0,0 +1,60 @@
+import * as path from 'path'
+import { pageWith } from 'page-with'
+import { waitUntil } from '../support/utils'
+
+function createRuntime() {
+  return pageWith({
+    example: path.resolve(__dirname, 'handle-stream.mocks.ts'),
+    routes(app) {
+      app.get('/stream', (req, res) => {
+        res.write('first-chunk')
+
+        setTimeout(() => {
+          res.write(' last-chunk')
+          res.end()
+        }, 1000 * 6)
+      })
+    },
+  })
+}
+
+test('should handle stream with no timeout', async () => {
```

Comment:
Nitpick: should the _stream_ have no timeout, or should MSW produce no timeouts?

Based on the code below, stream does have the `abortTimeout`, so I believe the test name can be:

```
test('handles a stream response without throwing a timeout error')
```

Reply:
yes the test name is better 👍